conf/application-json-logger.xml

```diff
@@ -5,7 +5,7 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="WARN"/>
+    <logger name="uk.gov" level="DEBUG"/>
```

Comment:
lets not commit these changes? both xml files

Reply:
Configured for local testing. Removed.